### ISSUE TITLE
Documentation: LdaModel.get_document_topics imprecise

### ISSUE THREAD

**Issue by johann-petrak** [RaRe-Technologies/gensim#2236]
Web documentation says:

"minimum_probability (float) – Topics with an assigned probability lower than this threshold will be discarded." 

Documentation string from my gensim version () says:

"minimum_probability (float): Ignore topics with probability below this value
        (None by default). If set to None, a value of 1e-8 is used to prevent 0s."

This sounds as if setting this parameter to 0.0 would be a guarantee to always get back as many tuples as there are topics in the model, but this is not the case: the code still enforces the minimum of 1e-8. 

I am also not sure what the point of this is: if the parameter is set to 0.0, why not allow the user to get back the full topic distribution? I would tend to make the method work as described, rather than change the description.

**Comment by menshikh-iv** [comment]
Hello, thanks for report @johann-petrak, I agree, documentation is not quite.

About `0.0` - I'm not sure, what's a reason. I'm +1 for returning "as many tuples as topics", wdyt @piskvorky?

**Comment by piskvorky** [comment]
Yeah, the docs could be improved. It lists three options in the `Return` section… which one is it then? How do I choose?

Values of 0.0 cannot be returned in **sparse** output. It would break various invariants and likely lead to segfaults.

If you want the full distribution (incl. zeroes), convert from sparse to dense with [gensim.matutils.sparse2full](https://radimrehurek.com/gensim/matutils.html#gensim.matutils.sparse2full).

**Comment by menshikh-iv** [comment]
@piskvorky 

> Values of 0.0 cannot be returned in sparse output. It would break various invariants and likely lead to segfaults.

Ok, but what's about values that non-zero but very small, like `1e-10`? What's a problem with it? With `sparse2full` we'll put `0.` instead of `1e-10`, because small values already filtered.

**Comment by piskvorky** [comment]
Yeah, tiny values are considered numerical artefacts (you cannot compare floats for exact equality), and treated the same as 0.0.
